### PR TITLE
[NETSTAT] Fix output formats and more

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Match Windows (XP).

JIRA issue: [CORE-16119](https://jira.reactos.org/browse/CORE-16119)

## TODO

- [ ] This PR is based on WXP. Someone else should check it on WS03.

### CODE REVIEW

File: base/applications/network/netstat/netstat.c [hunk 1]
```diff
@@ -169,6 +169,7 @@ BOOL DisplayOutput(VOID)
 {
     if (bNoOptions)
     {
+        ConResPuts(StdOut, IDS_ACTIVE_CONNECT);
```
Comment (SergeGautherie) [inline base/applications/network/netstat/netstat.c]: Added.

**Comment by SergeGautherie** [comment]
@Doug-Lyons, could you compare and report WS03 netstat outputs, on WS03? Thanks.

**Comment by SergeGautherie** [comment]
> `// FIXME: On WXP, 'Packets' has no 's'. Is ROS better and/or should it be identical?`

Fixed, as WS03 == WXP.

--

Also added bare WS03 `-t` option support.

**Comment by SergeGautherie** [comment]
@milawynsrealm, were your #1657 format changes on purpose or unintended?

File: base/applications/network/netstat/lang/en-US.rc [hunk 2]
```diff
@@ -16,9 +17,11 @@ NETSTAT [-a] [-e] [-n] [-o] [-p proto] [-r] [-s] [interval]\n\n\
   -s            Displays per-protocol statistics. By default, Statistics are\n\
                 shown for IP, ICMP, TCP and UDP;\n\
                 the -p option may be used to specify a subset of the default.\n\
+  -t            FIXME (Accepted option though unimplemented feature).\n\
+  -v            FIXME (Accepted option though unimplemented feature).\n\
```
Comment (learn-more) [inline base/applications/network/netstat/lang/en-US.rc]: Why add options not yet supported?

File: base/applications/network/netstat/lang/en-US.rc [hunk 3]
```diff
@@ -27,76 +30,76 @@ END
...
+    IDS_IP_OUT_REQUEST         "  Output Requests                    = %lu\n"
+    IDS_IP_ROUTE_DISCARD       "  Routing Discards                   = %lu\n"
+    IDS_IP_DISCARD_OUT_PACK    "  Discarded Output Packets           = %lu\n"
+    IDS_IP_OUT_PACKET_NO_ROUTE "  Output Packet No Route             = %lu\n"
+    IDS_IP_REASSEMBLE_REQUIRED "  Reassembly Required                = %lu\n"
+    IDS_IP_REASSEMBLE_SUCCESS  "  Reassembly Successful              = %lu\n"
+    IDS_IP_REASSEMBLE_FAILURE  "  Reassembly Failures                = %lu\n"
+    IDS_IP_DATAG_FRAG_SUCCESS  "  Datagrams Successfully Fragmented  = %lu\n"
+    IDS_IP_DATAG_FRAG_FAILURE  "  Datagrams Failing Fragmentation    = %lu\n"
+    IDS_IP_DATAG_FRAG_CREATE   "  Fragments Created                  = %lu\n"
```
Comment (learn-more) [inline base/applications/network/netstat/lang/en-US.rc]: Why the extra space in all lines?

File: base/applications/network/netstat/lang/en-US.rc [hunk 4]
```diff
@@ -27,76 +30,76 @@ END
...
-    IDS_TCP_SEG_SENT        "  Segments Sent              = %lu\n"
-    IDS_TCP_SEG_RETRANSMIT  "  Segments Retransmitted     = %lu\n"
+    IDS_TCP_ACTIVE_OPEN     "  Active Opens                        = %lu\n"
+    IDS_TCP_PASS_OPEN       "  Passive Opens                       = %lu\n"
+    IDS_TCP_FAIL_CONNECT    "  Failed Connection Attempts          = %lu\n"
+    IDS_TCP_RESET_CONNECT   "  Reset Connections                   = %lu\n"
+    IDS_TCP_CURRENT_CONNECT "  Current Connections                 = %lu\n"
+    IDS_TCP_SEG_RECEIVE     "  Segments Received                   = %lu\n"
+    IDS_TCP_SEG_SENT        "  Segments Sent                       = %lu\n"
+    IDS_TCP_SEG_RETRANSMIT  "  Segments Retransmitted              = %lu\n"
```
Comment (learn-more) [inline base/applications/network/netstat/lang/en-US.rc]: Why the extra space in all lines?

File: base/applications/network/netstat/lang/en-US.rc [hunk 5]
```diff
@@ -27,76 +30,76 @@ END
...
 BEGIN
     IDS_UDP_IP4_HEADER    "\nUDP Statistics for IPv4\n\n"
-    IDS_UDP_DATAG_RECEIVE "  Datagrams Received = %lu\n"
-    IDS_UDP_NO_PORT       "  No Ports           = %lu\n"
-    IDS_UDP_RECEIVE_ERROR "  Receive Errors     = %lu\n"
-    IDS_UDP_DATAG_SEND    "  Datagrams Sent     = %lu\n"
+    IDS_UDP_DATAG_RECEIVE "  Datagrams Received    = %lu\n"
+    IDS_UDP_NO_PORT       "  No Ports              = %lu\n"
+    IDS_UDP_RECEIVE_ERROR "  Receive Errors        = %lu\n"
+    IDS_UDP_DATAG_SEND    "  Datagrams Sent        = %lu\n"
```
Comment (learn-more) [inline base/applications/network/netstat/lang/en-US.rc]: Why the extra space in all lines?

File: base/applications/network/netstat/lang/en-US.rc [hunk 6]
```diff
@@ -27,76 +30,76 @@ END
...
-    IDS_ETHERNET_NON_UNICAST_PACKET "  Non-unicast packets %14lu %15lu\n"
-    IDS_ETHERNET_DISCARD            "  Discards            %14lu %15lu\n"
-    IDS_ETHERNET_ERROR              "  Errors              %14lu %15lu\n"
-    IDS_ETHERNET_UNKNOWN            "  Unknown Protocols   %14lu\n"
+    IDS_ETHERNET_BYTES              "Bytes                %14lu %15lu\n"
+    IDS_ETHERNET_UNICAST_PACKET     "Unicast packets      %14lu %15lu\n"
+    IDS_ETHERNET_NON_UNICAST_PACKET "Non-unicast packets  %14lu %15lu\n"
+    IDS_ETHERNET_DISCARD            "Discards             %14lu %15lu\n"
+    IDS_ETHERNET_ERROR              "Errors               %14lu %15lu\n"
+    IDS_ETHERNET_UNKNOWN            "Unknown protocols    %14lu\n"
```
Comment (learn-more) [inline base/applications/network/netstat/lang/en-US.rc]: Why the extra space in all lines?

File: base/applications/network/netstat/netstat.c [hunk 7]
```diff
@@ -119,16 +124,17 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                     case L's':
                         bDoShowProtoStats = TRUE;
                         break;
-                    case L'o':
-                        bDoShowProcessId = TRUE;
+                    case L't':
+                        // UNIMPLEMENTED.
+                        ConPuts(StdErr, L"'t' option is FIXME (Accepted option though unimplemented feature).\n");
```
Comment (learn-more) [inline base/applications/network/netstat/netstat.c]: Why?

File: base/applications/network/netstat/netstat.c [hunk 8]
```diff
@@ -119,16 +124,17 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
...
                         break;
                     case L'v':
-                        // FIXME!
-                        ConPuts(StdOut, L"got v\n");
+                        // UNIMPLEMENTED.
+                        ConPuts(StdErr, L"'v' option is FIXME (Accepted option though unimplemented feature).\n");
```
Comment (learn-more) [inline base/applications/network/netstat/netstat.c]: Why?

File: base/applications/network/netstat/lang/en-US.rc (hunk 2 above)
Comment (SergeGautherie) [reply base/applications/network/netstat/lang/en-US.rc]: To match code...

File: base/applications/network/netstat/lang/en-US.rc (hunk 3 above)
Comment (SergeGautherie) [reply base/applications/network/netstat/lang/en-US.rc]: As I already wrote, that matches XP output, and was changed without a word in a recent PR...

File: base/applications/network/netstat/netstat.c (hunk 7 above)
Comment (SergeGautherie) [reply base/applications/network/netstat/netstat.c]: (Why what?) Added to match other FIXME options...

File: base/applications/network/netstat/netstat.c (hunk 8 above)
Comment (SergeGautherie) [reply base/applications/network/netstat/netstat.c]: (Why what?) Changed what looks like a dev WIP to a more user friendly result.

File: base/applications/network/netstat/lang/en-US.rc (hunk 3 above)
Comment (HBelusca) [reply base/applications/network/netstat/lang/en-US.rc]: Is the aligning whitespace size the same in other localizations?

Comment (SergeGautherie) [reply base/applications/network/netstat/lang/en-US.rc]: I have no idea: ROS has EN-US only, I compared to EN-US WXP (only)...

File: base/applications/network/netstat/netstat.c (hunk 8 above)
Comment (learn-more) [reply base/applications/network/netstat/netstat.c]: This does not like a comment meant for a user....

Comment (SergeGautherie) [reply base/applications/network/netstat/netstat.c]: I think it is an improvement over current code.
Feel free to suggest something even better...

**Pull request merged**